website/docs/getting-started/index.mdx

```diff
@@ -157,36 +149,38 @@ const Posts = () => {
   </div>
 </template>
 
-<script>
+<script lang="ts">
```

Comment:
i think lang was missing, after adding this prettier prettified content inside `script` tag